Firebase/CoreDiagnostics/FIRCDLibrary/FIRCoreDiagnostics.m

```diff
@@ -149,66 +158,30 @@ + (instancetype)sharedInstance {
 }
 
 - (instancetype)init {
+  GDTTransport *transport = [[GDTTransport alloc] initWithMappingID:@"137"
+                                                       transformers:nil
+                                                             target:kGDTTargetCCT];
+
+  FIRCoreDiagnosticsDateFileStorage *dateStorage = [[FIRCoreDiagnosticsDateFileStorage alloc]
+      initWithFileURL:[[self class] filePathURLWithName:kFIRDiagnosticsHeartbeatDateFileName]];
+
+  return [self initWithTransport:transport heartbeatDateStorage:dateStorage];
+}
+
+/** Initializer for unit tests */
```

Comment:
Please fix this docstring--periods after sentences, two missing ATparams, one missing ATreturn.

Reply:
Updated the doc string